### ISSUE TITLE
Training Parameters

### ISSUE THREAD

**Issue by KhushiGala** [tkim-snu/GLACNet#6]
Can you please share the number of epochs that you trained the model for, and what were the values of the other parameters to obtain the pretrained model that you have currently provided? After which epoch did the model start overfitting and what were the corresponding loss values?
